ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.txt

```diff
@@ -0,0 +1,231 @@
+1145.1 "Lunge 1" sync /:Knave Of Hearts:5EB1:/
+1146.2 "--sync--" sync /:Knave Of Hearts:5EB0:/
+1155.1 "Lunge 2" sync /:Copied Knave:5EB1:/
+1167.3 "Roar" sync /:Knave Of Hearts:5EB5:/
+
+1178.5 "Replicate" sync /:Knave Of Hearts:5EA9:/
+1189.8 "Stacking The Deck" sync /:Copied Knave:60F0:/
+1200.0 "Colossal Impact 1" sync /:Knave Of Hearts:5EA[47]:/
+1206.0 "Colossal Impact 2" sync /:Copied Knave:5EA[47]:/
+1208.1 "Spheroids" sync /:Knave Of Hearts:5EA8:/
+1211.0 "Colossal Impact 3" sync /:Copied Knave:5EA[47]:/
+1213.2 "--sync--" sync /:Knave Of Hearts:5EAA:/ # Magic Artillery Alpha cast
+1221.2 "Magic Barrage 1" #sync /:Spheroid:5EAC:/
+1221.3 "Magic Artillery Alpha" sync /:Knave Of Hearts:5EAB:/ # damage
+1224.2 "Magic Barrage 2" #sync /:Spheroid:5EAC:/
+1239.4 "Light Leap" sync /:Knave Of Hearts:5EAE:/
+1239.4 "--untargetable--"
+1241.5 "--sync--" sync /:Knave Of Hearts:5EB0:/
+1243.5 "--targetable--"
+
+1246.6 "Replicate" sync /:Knave Of Hearts:5EA9:/
+1257.9 "Stacking The Deck" sync /:Copied Knave:60F0:/
+1268.1 "Lunge 1" sync /:Knave Of Hearts:5EB1:/
+1269.2 "--sync--" sync /:Knave Of Hearts:5EB0:/
+1278.1 "Lunge 2" sync /:Copied Knave:5EB1:/
+1280.1 "Colossal Impact" sync /:Knave Of Hearts:60C8:/
+1287.2 "--sync--" sync /:Knave Of Hearts:5EB2:/ # Magic Artillery Beta cast
+1294.3 "Magic Artillery Beta" sync /:Knave Of Hearts:5EB3:/ # damage
+
+1305.4 "Spheroids" sync /:Knave Of Hearts:5EA8:/
+1309.6 "Knavish Bullets" sync /:Spheroid:5EAD:/
+1318.3 "Colossal Impact" sync /:Knave Of Hearts:5EA7:/
+1332.5 "Roar" sync /:Knave Of Hearts:5EB5:/
+
+# loop
+1342.7 "Replicate" sync /:Knave Of Hearts:5EA9:/ window 50,50 jump 1178.5
+1354.0 "Stacking The Deck" #sync /:Copied Knave:60F0:/
+1364.2 "Colossal Impact 1" #sync /:Knave Of Hearts:5EA7:/
+1370.2 "Colossal Impact 2" #sync /:Copied Knave:5EA7:/
+1372.3 "Spheroids" #sync /:Knave Of Hearts:5EA8:/
+1375.2 "Colossal Impact 3" #sync /:Copied Knave:5EA4:/
+1377.4 "--sync--" #sync /:Knave Of Hearts:5EAA:/ # Magic Artillery Alpha cast
+1385.4 "Magic Barrage 1" #sync /:Spheroid:5EAC:/
+1385.5 "Magic Artillery Alpha" sync /:Knave Of Hearts:5EAB:/ # damage
+1388.4 "Magic Barrage 2" #sync /:Spheroid:5EAC:/
+1403.6 "Light Leap" #sync /:Knave Of Hearts:5EAE:/
+1403.6 "--untargetable--"
+1405.7 "--sync--" #sync /:Knave Of Hearts:5EB0:/
+1407.7 "--targetable--"
+
+### Hansel & Gretel
+# -p 5C68:2012 (if Hansel starts with shield)
+# -p 5C69:2012 (if Gretel starts with shield)
+# -ii 5CEC 5CED 5CEE 5CEF 5C70 5C67 5C5C 5C6C 56CD 5C7F 5C55 61B9 61DB 5C65 5C66 5C6E 5C6F 5C57 5CF2 61BD 5C5B 5C72 61BB 55B3 5C5D 5C59 5C7D
+# -ic "2B"
+# -it "Hansel"
+
+2000.0 "--sync--" sync /00:0839:Staging Node B will be sealed off/ window 10000,0
+2012.0 "Upgraded Shield" sync /:Gretel:5C6[89]:/
+2012.0 "Upgraded Lance" sync /:Hansel:5C6[AB]:/
+2025.2 "Wail" sync /:Hansel:5C77:/
+2025.3 "Crippling Blow (G)" sync /:Gretel:5C78:/
+2035.5 "Wail" sync /:Gretel:5C76:/
+2035.6 "Crippling Blow (H)" sync /:Hansel:5C79:/
```

Comment:
Are these always in this order?

Reply:
The crippling blows were always in the same order from the same mobs, if that's what you're asking.  Maybe @JLGarber can confirm from his log?

Reply:
My run saw the same thing, yes. Every Wail was Hansel first, every Crippling Blow was Gretel first.